std/io/streams.ts

```diff
@@ -34,3 +34,31 @@ export function fromStreamReader(
     },
   };
 }
+
+/** Create a `WritableStream` from a `Writer`. */
+export function writableStreamFromWriter(
+  writer: Deno.Writer,
+): WritableStream<Uint8Array> {
+  return new WritableStream({
+    async write(chunk) {
+      await Deno.writeAll(writer, chunk);
+    },
+  });
+}
+
+/** Create a `ReadableStream` from an `AsyncIterator`. */
+export function readableStreamFromAsyncIterator<T>(
+  iterator: AsyncIterableIterator<T>,
+): ReadableStream<T> {
```

Comment:
IMO `readableStreamFromReader()` should exist either way. This module exists to provide bridges between Deno IO and whatwg streams. One should be able to "expect" a `readableStreamFromReader` looking at the other functions.

In this case, there's no problem with having both even if one would have a trivial implementation. Similar to `readLines()` in `std/io/bufio.ts`.

Reply:
@nayeemrmn i will look into it soon; the best usage for that would a bytes ReadableStream, but still not knowledgeable about those